conversion/requests/duckgo/convert.go

```diff
@@ -13,6 +13,14 @@ func ConvertAPIRequest(api_request officialtypes.APIRequest) duckgotypes.ApiRequ
 	if strings.HasPrefix(strings.ToLower(api_request.Model), "claude") {
 		duckgo_request.Model = "claude-3-haiku-20240307"
 	}
+	if strings.HasPrefix(strings.ToLower(api_request.Model), "Llama-3-70b") {
```

Comment:
It seems that using `ToLower(model)` here makes it impossible to match string with a capital letter.

Reply:
Can you solve this problem?